databases/week1/MySqlLesson1HomeWork.sql

```diff
@@ -0,0 +1,35 @@
+
+-- #1 Find out how many tasks are in the task table
+SELECT COUNT(id) AS task_count FROM task;
+
+-- #2 Find out how many tasks in the task table do not have a valid due date
+ SELECT COUNT(id) AS invalid_due_date FROM task WHERE task.due_date IS NULL;
+ 
+ -- #3 Find all the tasks that are marked as done
+ SELECT task.title,status.name FROM task INNER JOIN status ON status.name ='done';
+ 
+ -- #4 Find all the tasks that are not marked as done
+ SELECT task.title,status.name FROM task INNER JOIN status ON status.name !='done';
```

Comment:
You are missing a few things in these two queries before they will return the correct results. Can you spot what's missing?